test/fixtures/cookbooks/test/recipes/sup.rb

```diff
@@ -8,6 +8,6 @@
 
 ruby_block 'wait-right-here' do
   block do
-    sleep 5
+    sleep 10
```

Comment:
Were we not sleepy enough?

Reply:
not sleepy enough when I was running the tests locally with vagrant, no.